src/aead/gcm.rs

```diff
@@ -292,6 +298,18 @@ enum Implementation {
     Fallback,
 }
 
+#[cfg(any(target_arch = "mips64", target_arch = "mips"))]
+#[inline]
+fn detect_implementation(_cpu: cpu::Features) -> Implementation {
+    Implementation::Fallback
+}
```

Comment:
Instead of doing this, change cpu.rs so that you can construct a no-op `cpu::Features` on MIPS. In fact, I think I already did that for wasm32, so I think you can just delete all of these changes.